qualtran/bloqs/basic_gates/cnot.py

```diff
@@ -117,7 +117,7 @@ def add_controlled(
                 (c1, c2), target = bb.add(
                     bloq, ctrl=np.array([in_soqs['ctrl'], c2]), target=in_soqs['target']
                 )
-                return (c1,), (c2, target)
```

Comment:
It's confusing that the ctrl is ordered with the new control "below" the original cnot control, or am I being stupid?

Reply:
It shouldn't matter as long as it's consistent -- which it wasn't. but now it is!

Reply:
does the diagram look correct, like if it was off-on-trg, vs on-off-trg

Reply:
I changed it so that the "new" control bit is the first one. In either case, the diagram is topologically correct. Whether it shows up in the order you expect I couldn't say